Sources/Jetworking/Client/Executor/UploadExecutor/UploadHandler.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+
+/// The handler used to combine progress and completion.
+public struct UploadHandler {
+    /// Typealias for the progress handler which returns the total bytes sent and the total bytes expected to be sent.
+    public typealias ProgressHandler = ((Int64, Int64) -> Void)?
```

Comment:
Maybe we could use the same handler for Uploading and downloading?

Reply:
I would like to separate the concerns here. Yes you are right the progress handler is the same but logically speaking we have two different use cases here. Also when at some point any differences between upload and download occur we have to introduce a breaking change which might not be understandable.